blazarclient/v1/resources.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) 2019 StackHPC Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from blazarclient import base
+from blazarclient.i18n import _
+
+import logging
+LOG = logging.getLogger(__name__)
+
+class ResourceClientManager(base.BaseClientManager):
+    def list_resources(self, sort_by=None):
+        resp, body = self.request_manager.get('/resources')
+        if sort_by:
+            resources = sorted(body, key=lambda l: l[sort_by])
```

Comment:
This variable is unused, and `sorted` produces a copy, so nothing will actually be sorted to the user.